exercises/concept/secrets/tests/Tests.elm

```diff
@@ -0,0 +1,52 @@
+module Tests exposing (tests)
+
+import Expect
+import Secrets exposing (..)
```

Comment:
It's a bit more verbose, but I think the tests are more readable when the functions are qualified.